src/devices/video/gb_lcd.cpp

```diff
@@ -3819,40 +3847,34 @@ void sgb_ppu_device::sgb_io_write_pal(int offs, uint8_t *data)
 		case 0x10:  /* DATA_TRN */
 			/* Not Implemented */
 			break;
+		case 0x11:  /* MLT_REQ */
+			/* MLT_REQ currently handled inside gb.cpp logic */
+			break;
 		case 0x12:  /* JUMP */
 			/* Not Implemented */
 			break;
 		case 0x13:  /* CHR_TRN */
-			if (data[1] & 0x1)
-				memcpy(m_sgb_tile_data.get() + 4096, m_vram.get() + 0x0800, 4096);
+			if (data[1] & 0x01u)
```

Comment:
I actually find the BIT macro makes a lot of code much harder to read as you're having to convert back and forth between what the game writes and the hex values in your head every time, and in multi-bit situations they're even worse.  I think this one should be up to whoever is writing the code.